Userland/Libraries/LibWeb/CSS/SelectorEngine.cpp

```diff
@@ -351,14 +351,14 @@ static inline bool matches(CSS::Selector::SimpleSelector const& component, DOM::
     case CSS::Selector::SimpleSelector::Type::Universal:
         return true;
     case CSS::Selector::SimpleSelector::Type::Id:
-        return component.name() == element.attribute(HTML::AttributeNames::id);
+        return component.name().bytes_as_string_view() == element.attribute(HTML::AttributeNames::id);
```

Comment:
I think the to-view method should be on the RHS:
```c++
return component.name() == element.attribute(HTML::AttributeNames::id).view();
```

This will make it clearer that the LHS is as it should be, and it's the RHS that needs to change in the future.